tests/check_framework/test_asynchronous.py

```diff
@@ -0,0 +1,16 @@
+import os
+from unittest import mock
+
+from django.core.checks import asynchronous
+from django.test import SimpleTestCase
+
+
+class CheckAsynchronousSafetyTest(SimpleTestCase):
+    @property
+    def func(self):
+        from django.core.checks.asynchronous import check_async_unsafe
+        return check_async_unsafe
```

Comment:
Oh yeah, you don't need to copy this weird property import pattern that the other checks tests do. I've been meaning to undo it. You can just import it directly.

Reply:
Ok.

Reply:
Done in https://github.com/django/django/pull/12601